clang/www/get_started.html

```diff
@@ -74,8 +74,8 @@ <h3 id="buildNix">On Unix-like Systems</h3>
         <a href="https://llvm.org/docs/CMake.html#frequently-used-cmake-variables">frequently used cmake variables</a>
         for more options.
     </li>
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 72 to 76]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->I think we should update this text at the same time to add something like:
```
This example uses the <a href="https://ninja-build.org/">Ninja build system</a>, but see <a href="https://cmake.org/cmake/help/latest/manual/cmake-generators.7.html"CMake's documentation about generators</a> for more options.
```
<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/llvm/llvm-project/93503?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->